ci/build_linux_wheels.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Host-side wrapper: workflow provides only --arch.
+Images are defined as regular Python lists (no env vars).
+
+Environment:
+  - GITHUB_WORKSPACE (optional; defaults to cwd)
+"""
+from __future__ import annotations
+import argparse
+import os
+import shlex
+import subprocess
+import sys
+from typing import List
+
+SCRIPT = r'''set -e
+yum install -y git sudo wget || true
+
+git config --global --add safe.directory /work
+
+# Determine Python versions to test
+if [ "$RELEASE" = "1" ]; then
+    PYTHON_VERSIONS="cp38-cp38 cp39-cp39 cp310-cp310 cp311-cp311 cp312-cp312 cp313-cp313"
+else
+    PYTHON_VERSIONS="cp38-cp38 cp313-cp313"
+fi
+
+ci/run_ci.sh install_bazel
+export PATH="$HOME/.local/bin:$PATH"
+
+# use the python interpreters preinstalled in manylinux
+OLD_PATH=$PATH
+for PY in $PYTHON_VERSIONS; do
+    export PYTHON_PATH="/opt/python/$PY/bin/python"
+    export PATH="/opt/python/$PY/bin:$OLD_PATH"
+    echo "Using $PYTHON_PATH"
+    python -m pip install cython wheel pytest
+    ci/deploy.sh build_pyfory
+
+    latest_wheel=$(ls -t dist/*.whl | head -n1)
+    echo "Attempting to install $latest_wheel"
+    python -m pip install "$latest_wheel"
+    python -c "import pyfory; print(pyfory.__version__)"
+
+    bazel clean --expunge
+done
+export PATH=$OLD_PATH
+'''
+
+DEFAULT_X86_IMAGES = [
+    "quay.io/pypa/manylinux2014_x86_64:latest",
+    # "quay.io/pypa/manylinux_2_28_x86_64:latest",
```

Comment:
Do we need to also build wheel for `manylinux_2_28_x86_64`? pyarrow released both: https://pypi.org/project/pyarrow/17.0.0/#files

Reply:
If so, I think we can release both wheels. Just not sure if both are released, which one will pip choose

Reply:
I would benchmark them before doing that.

Reply:
OK, we can do that later